src/index.mjs

```diff
@@ -70,14 +60,29 @@ function isIgnored(node, filter) {
  * @param {Array|RegExp|Function} options.ignores - Custom filter(s) that run after origin checks
  * @param {Number} options.timeout - Timeout after which prefetching will occur
  * @param {Function} options.timeoutFn - Custom timeout function
+ * @param {Object} options.config - IntersectionObserver configuration options
+ * @param {Element} options.config.root - The root container in which elements are observed
+ * @param {string} options.config.rootMargin - Margin added to root before intersection observation testing
+ * @param {number} options.config.threshold - Percentage of observed element in root before IO callback is called
  */
 export default function (options) {
   options = Object.assign({
     timeout: 2e3,
     priority: false,
     timeoutFn: requestIdleCallback,
     el: document,
+    config: {}
```

Comment:
We don't need this line, as `new IntersectionObserver(noop, undefined)` is still valid 😆 
```suggestion
```